Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -20,23 +23,27 @@ void Awake()
     private LayerMask NPCMask;
     private int maxMana = 10;
     private int currentMana;
-    
     private IInteractable interactable;
-    
+
+   // public OverlayManager overlayManager;
     protected new void Start()
     {
-        NPCMask= LayerMask.GetMask("NPC");
-        name = "Player";
-        maxHealth = 15;
-        base.Start();
-        healthBar.SetMaxHealth(maxHealth);
-        currentMana = maxMana;
-        manaBar.SetMaxMana(maxMana);
-    }
+        //gameManager = gameObject.AddComponent<OverlayManager>();
+        OverlayManager.FinalRoomScore = 6;
```

Comment:
Since we don't have a defined way of tracking progress yet, can we get rid of this for now?

For when we do end up tracking such values, it should happen somewhere else, not in PlayerController

Reply:
No